sample-templates/query-assist-data-summary-agent-claude-tested.json

```diff
@@ -0,0 +1,94 @@
+{
+    "name": "Query Assist Data Summary Agent",
+    "description": "Create Query Assist Data Summary Agent using Claude on BedRock",
+    "use_case": "REGISTER_AGENT",
+    "version": {
+      "template": "1.0.0",
+      "compatibility": ["2.17.0", "3.0.0"]
+    },
+    "workflows": {
+      "provision": {
+        "user_params": {},
+        "nodes": [
+          {
+            "id": "create_claude_connector",
+            "type": "create_connector",
+            "previous_node_inputs": {},
+            "user_inputs": {
+              "version": "1",
+              "name": "Claude instant runtime Connector",
+              "protocol": "aws_sigv4",
+              "description": "The connector to BedRock service for Claude model",
+              "actions": [
+                {
+                  "headers": {
+                    "x-amz-content-sha256": "required",
+                    "content-type": "application/json"
+                  },
+                  "method": "POST",
+                  "request_body": "{\"prompt\":\"${parameters.prompt}\", \"max_tokens_to_sample\":${parameters.max_tokens_to_sample}, \"temperature\":${parameters.temperature},  \"anthropic_version\":\"${parameters.anthropic_version}\" }",
+                  "action_type": "predict",
+                  "url": "https://bedrock-runtime.us-west-2.amazonaws.com/model/anthropic.claude-instant-v1/invoke"
+                }
+              ],
+              "credential": {
+                  "access_key": "<YOUR_ACCESS_KEY>",
+                  "secret_key": "<YOUR_SECRET_KEY>",
+                  "session_token": "<YOUR_SESSION_TOKEN>"
+              },
```

Comment:
Not a comment on this PR, but note to other reviewers. I wonder if there's a way we can (probably in ML Commons) not require these to be specified in the connector, but instead have them pulled in real time using CredentialsProviders like the AWS SDK client does.